src/Memo/Creator/MemoCreatorFactory.php

```diff
@@ -22,6 +22,8 @@ public static function create(Table $table)
             case TableType::DBASE_7_MEMO:
                 return new DBase7MemoCreator($table);
             //todo foxpro
```

Comment:
Should we delete this?

Reply:
The `todo`? That I'm not sure, actually. I don't know anything about the other FoxPro versions and I've found that documentation is a bit scarce. I'm not sure if my changes here will work properly for the other versions :/